### PR TITLE
test: review skipped tests

### PR DESCRIPTION
**Pull request by mavmaf**
## Motivation and context

In the past, CI Cypress logs were not being displayed and some tests were skipped as a misinterpretation of this error. Now that the problem was fixed in [EMP-503](https://searchbroker.atlassian.net/browse/EMP-503). It's time to review if the deactivated tests need to be refactor.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [x] `Main`
- [ ] Other. Specify:

## How has this been tested?

Run the build, then run it again, and again until you get tired or see a failing test.

Tests performed according to [testing guidelines](./contributing/tests.md): 

## Checklist:

- [ ] My code follows the **[style guidelines](./CONTRIBUTING.md#style-guides)** of this project.
- [ ] I have performed a **self-review** on my own code.
- [ ] I have **commented** my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the **[documentation](./CONTRIBUTING.md#documentation-style-guide)**.
- [ ] My changes generate **no new warnings**.
- [ ] I have added **tests** that prove my fix is effective or that my feature works.
- [ ] New and existing **unit tests pass locally** with my changes.

[EMP-503]: https://searchbroker.atlassian.net/browse/EMP-503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/x-components/tests/e2e/history-queries.feature
Comment (CachedaCodes) [inline packages/x-components/tests/e2e/history-queries.feature:1]: Test goes from Scenario Outline 1 to 3

File: packages/x-components/tests/e2e/history-queries.feature [hunk 1]
```diff
@@ -7,35 +7,25 @@ Feature: History queries component
...
-
-  Scenario Outline: 2. History query delete button is clicked
-    Given following config: hide if equals query <hideIfEqualsQuery>, debounce <debounceInMs>, requested items <maxItemsToStore>, rendered <maxItemsToRender>, instant search <instant>
-    And   start button is clicked
-    And   a "<list>" of queries already searched
-    When  search-input is focused
+    When  clear search button is pressed
     And   the delete button of <historyQueryItem> is clicked
     Then  the deleted history query is removed from history queries
     And   the number of rendered history queries is <maxItemsToRender> - 1 if <maxItemsToStore> < <maxItemsToRender>
```
Comment (CachedaCodes) [inline packages/x-components/tests/e2e/history-queries.feature:24]: IDE is complaining about this line (says that the step is not defined, but I can see it defined, maybe it's nothing
)

File: packages/x-components/tests/e2e/popular-searches/popular-searches.spec.ts [hunk 2]
```diff
@@ -27,13 +27,6 @@ Given(
...
 );
 
-// Scenario 1
-Then('at most {int} popular searches are displayed', (maxItemsToRender: number) => {
-  cy.getByDataTest('popular-search')
-    .should('have.length.at.least', 1)
-    .and('have.length.at.most', maxItemsToRender);
-});
-
 // Scenario 2
```
Comment (CachedaCodes) [inline packages/x-components/tests/e2e/popular-searches/popular-searches.spec.ts:30]:
```suggestion
// Scenario 1
```

File: packages/x-components/tests/e2e/history-queries.feature (hunk 1 above)
Comment (mavmaf) [reply packages/x-components/tests/e2e/history-queries.feature:24]: It is defined on `history-queries.spec.ts`, line 81 onwards. For whatever reason it doesn't like `<` symbol... I can change it but we already have the same issue for the steps with a boolean parameter on them, so... let me know :)

File: packages/x-components/tests/e2e/popular-searches/popular-searches.spec.ts (hunk 2 above)
Comment (mavmaf) [reply packages/x-components/tests/e2e/popular-searches/popular-searches.spec.ts]: It is in fact Scenario 2, as Scenario 1 steps are defined in `common-steps.spec.ts`

**Review by CachedaCodes — APPROVED** [review]

**Pull request merged**